napari/utils/colormaps/standardize_color.py

```diff
@@ -63,6 +63,29 @@ def transform_color(colors: Any) -> np.ndarray:
     return _color_switch[colortype](colors)
 
 
+def transform_single_color(color):
```

Comment:
I feel like this came up in the past... but this makes me a little sad (having a new function just to pull out the first index of transform_color...)  is there no cleaner way to have it just be `transform_color`?

Reply:
hmm i'll give it a try, def get what you mean

Reply:
if i end up looking across PRs I do want to use this in a couple places, and i couldn't think of a cleaner way to do this. i can def live with this for now and then maybe we improve at some point if we have a better idea